Admin/BaseMediaAdmin.php

```diff
@@ -106,7 +106,7 @@ public function getNewInstance()
 
         if ($this->hasRequest()) {
             if ($this->getRequest()->isMethod('POST')) {
-                $media->setProviderName($this->getRequest()->get(sprintf('%s[providerName]', $this->getUniqid()), null, true));
+                $media->setProviderName($this->getRequest()->get(sprintf('%s', $this->getUniqid()), null, true)['providerName']);
```

Comment:
I think you meant http-foundation since ```$this->getRequest()``` returns a ParameterBag object which belongs to httpfoundation.
2.8: https://github.com/symfony/http-foundation/blob/2.8/ParameterBag.php#L91
3.1: https://github.com/symfony/http-foundation/blob/3.1/ParameterBag.php#L86
I believe the parameter of deep = true in this case already triggers an error if the app is using http-foundation <=2.8. 

Then Why don't we just remove them all and the problem will be solved without extra checks ?